### PR TITLE
feat: update maintainers.yml when creating new service

### PR DESCRIPTION
**Pull request by mydiemho**
resolves component of https://github.com/microsoft/bedrock/issues/577

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -91,3 +91,6 @@ typings/
 
 # DynamoDB Local files
 .dynamodb/
+
+# vscode
+.vscode
```
Comment (evanlouie) [inline .gitignore]: newline plz 🙏

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        "--runInBand"
+      ],
+      "cwd": "${workspaceFolder}",
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen",
+      "disableOptimisticBPs": true,
+      "program": "${workspaceFolder}/node_modules/jest/bin/jest"
+    }
+  ]
+}
```
Comment (evanlouie) [inline .vscode/launch.json]: newline plz 🙏

File: package.json [hunk 3]
```diff
@@ -48,10 +48,13 @@
     ]
   },
   "dependencies": {
+    "@types/mock-fs": "^3.6.30",
```
Comment (evanlouie) [inline package.json]: Keep type definitions in `devDependencies`; run `yarn remove @types/mock-fs` and then `yarn add -D @types/mock-fs`

**Review by evanlouie — CHANGES_REQUESTED** [review]

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,10 @@
+root = true  ; top-most EditorConfig file
```
Comment (mydiemho) [inline .editorconfig:1]: @evanlouie  are you ok with adding this file?

File: src/commands/service/create.test.ts [hunk 5]
```diff
@@ -50,3 +96,31 @@ describe("Adding a service to a repo directory", () => {
     // TODO: Verify root project bedrock.yaml and maintainers.yaml has been changed too.
   });
 });
+
+const writeSampleMaintainersFileToDir = async (maintainersFilePath: string) => {
+  const content = {
```
Comment (evanlouie) [inline src/commands/service/create.test.ts]: add a typing of `const content: IMaintainersFile = {`

File: .editorconfig (hunk 4 above)
Comment (evanlouie) [reply .editorconfig:1]: Should be okay. Don't think it conflicts with any of the prettier configs we have

**Review by evanlouie — CHANGES_REQUESTED** [review]

**Comment by mtarng** [comment]
Not sure what's going on with the `yarn.lock` conflict though... looks fine on my machine

**Review by evanlouie — APPROVED** [review]

**Review by bnookala — APPROVED** [review]

**Pull request merged**